lib/determine-basal/determine-basal.js

```diff
@@ -922,8 +922,8 @@ var maxDelta_bg_threshold;
 
     // if not in LGS mode, cancel temps before the top of the hour to reduce beeping/vibration
     // console.error(profile.skip_neutral_temps, rT.deliverAt.getMinutes());
-    if ( profile.skip_neutral_temps && rT.deliverAt.getMinutes() >= 55 ) {
-        rT.reason += "; Canceling temp at " + rT.deliverAt.getMinutes() + "m past the hour. ";
+    if ( profile.skip_neutral_temps && !enableSMB && rT.deliverAt.getMinutes() >= 55 ) {
```

Comment:
Do we want to print some sort of “ignoring skip_neutral_temps because SMB is enabled” message if enableSMB?

Reply:
done that, but only for the log, no rT.reason output - thinking too much clutter
```
2023-06-27 07:56:17.653895+0200 DEV: JavaScript log: maxDelta threshold for BG-Jump to allow SMB's set to: 30%
2023-06-27 07:56:17.654444+0200 DEV: JavaScript log: BG projected to remain above 100 for 240minutes
2023-06-27 07:56:17.654994+0200 DEV: JavaScript log: naive_eventualBG: 120, bgUndershoot: -50, zeroTempDuration: 240, zeroTempEffect: 351, carbsReq: -94
2023-06-27 07:56:17.655532+0200 DEV: JavaScript log: 4min before turn of the hour, but SMB's are enabled - no skipping neutral temps
2023-06-27 07:56:17.656076+0200 DEV: JavaScript log: SMB max range extended from default by factor 3
2023-06-27 07:56:17.656608+0200 DEV: JavaScript log: profile.maxSMBBasalMinutes:,120,profile.current_basal:,2.05
2023-06-27 07:56:17.657135+0200 DEV: JavaScript log: SMB Delivery Ratio increased from default 0.5 to 0.68
2023-06-27 07:56:17.657669+0200 DEV: JavaScript log: naive_eventualBG 120, 0m 0U/h temp needed; last bolus 1139.9m ago; maxBolus: 12.3
```